features/feature_case/driverMem/driverMem.cu

```diff
@@ -9,7 +9,10 @@
 #include <cuda.h>
 #include <cuda_runtime_api.h>
 #include <iostream>
-int main(){
+#include <vector>
+#include <algorithm>
+
+void test1(){
```

Comment:
Is test1 kept only to ensure that these calls are migrated and that they compile?  I don't see it being called anywhere. In the previous version this migrated test1 code was executed, but the results were not checked.  I think it would be good to still execute them.  The previous version would have caught a crash in the test1 code, but the new version will not.

Reply:
All the code wasn't being ran before, as the test wasn't in the `exec_tests` list in `test_feature.py`, so the test driver was only compiling the code. The code before migration segfaults, so since I was trying to make a test for `cuMemcpy`, I put the previous code in another function that isn't ran to still make sure it compiles.